packages/cardpay-sdk/sdk/utils/merchant.ts

```diff
@@ -0,0 +1,45 @@
+import { isAddress } from 'web3-utils';
+
+export function validateMerchantId(value: string) {
+  if (!value.trim().length) return 'This field is required';
+  if (/[^a-z0-9]/.test(value))
+    return 'The Merchant ID can only contain lowercase letters or numbers, no special characters';
+  else if (value.length > 50)
```

Comment:
Let's at least name this magic number. Is anything else using this "50" number?

Reply:
Nope. This is basically a round number < 63, which is the domain name limit.